src/css/Parser.js

```diff
@@ -1986,6 +1986,11 @@ Parser.prototype = function() {
                     functionText = tokenStream.token().value;
                     this._readWhitespace();
                     expr = this._expr(true);
+                    if (expr === null) {
+                      // the FUNCTION is set if there's a ( . It doesn't check for a closing ) in identOrFunctionToken in TokenStream..
+                      // if there's nothing between the brackets, expr is null
+                      throw new SyntaxError("Expected an expression in the function on line " + tokenStream.token().startLine + ", col " + tokenStream.token().startCol + ".", tokenStream.token().startLine, tokenStream.token().startCol);
```

Comment:
You don't need to emit the line number message yourself; all the other SyntaxErrors in this code just pass them as the second and third arguments. Following that practice will simplify this line a lot.